components/inputs/input-time.js

```diff
@@ -91,14 +115,45 @@ class InputTime extends LitElement {
 	}
 
 	render() {
+		if (INTERVALS === null) {
```

Comment:
Can you put this check in the `getIntervals` method itself? That way it's the one gating access to the variable, instead of every place that needs the intervals (which is just render right now).